proto/dydxprotocol/indexer/off_chain_updates/off_chain_updates.proto

```diff
@@ -88,13 +88,25 @@ message OrderUpdateV1 {
   uint64 total_filled_quantums = 2;
 }
 
+// OrderReplace messages contain the replacement order.
+message OrderReplaceV1 {
+  dydxprotocol.indexer.protocol.v1.IndexerOrder order = 1;
+  dydxprotocol.indexer.off_chain_updates.OrderPlaceV1.OrderPlacementStatus
```

Comment:
Do we want new enum values for replacement? Feel free to handle in different PR.

Reply:
I'm not sure, do you see a use case for new values?

Reply:
to distinguish between placed vs replaced orders.

Reply:
Discussed offline, will leave this for now. We can distinguish based on message type